src/main/java/org/folio/note/NoteServiceImpl.java

```diff
@@ -91,4 +97,8 @@ private UserDisplayInfo getUserDisplayInfo(String firstName, String middleName,
     userDisplayInfo.setLastName(lastName);
     return userDisplayInfo;
   }
+
+  private String sanitizeHtml(String content) {
+    return Jsoup.clean(content, Whitelist.relaxed().removeTags("img"));
```

Comment:
@andrii-paias : want to understand this better - what does this "Whitelist.relaxed().removeTags("img")" do? Thanks!

Reply:
Just curious -- Is there anyway that we can know or log the fact that the note HTML content has been revised because it has been sanitized?

Reply:
@sduvvuri-ebsco, 
Whitelist.relaxed() is a default set of allowed tags:
"a", "b", "blockquote", "br", "caption", "cite", "code", "col", "colgroup", "dd", "div", "dl", "dt", "em", "h1", "h2", "h3", "h4", "h5", "h6", "i", "li", "ol", "p", "pre", "q", "small", "span", "strike", "strong", "sub", "sup", "table", "tbody", "td", "tfoot", "th", "thead", "tr", "u", "ul"

I removed tag <img> from this list, because folio WYSIWYG doesn't support images, and <img> can inline content from external sources, all other tags are just for formatting text + there are tags related to tables

Reply:
@carolegodfrey, 
Thanks, added logging of sanitizied result